types/iobroker/index.d.ts

```diff
@@ -514,10 +536,36 @@ declare global {
             sendToHostAsync(hostName: string, message: MessagePayload): Promise<Message | undefined>;
             sendToHostAsync(hostName: string, command: string, message: MessagePayload): Promise<Message | undefined>;
 
+            /**
+             * Send a notification with the given scope and category to the host of this adapter
+             * @param scope The scope of the notification
+             * @param category The category of the notification. If `null` is passed, ioBroker will try to infer a matching category from the `message` parameter.
+             * @param message The message of the notification
+             */
+            registerNotification<Scope extends keyof NotificationScopes>(
+                scope: Scope,
+                category: NotificationScopes[Scope] | null,
+                message: string,
+            ): Promise<void>;
+
+            /**
+             * Set capabilities of the given executable. Only works on Linux systems.
+             * @param execPath Path to the executable. For Node.js, this can be determined it via `process.execPath`.
```

Comment:
the `it` seems a typo, probably in controller too

Reply:
I reworded that a bit, because it had no punctuation whatsoever.